DependencyInjection/Configuration.php

```diff
@@ -153,8 +153,9 @@ public function getConfigTreeBuilder()
 
     private function configureClients(ArrayNodeDefinition $root)
     {
-        $root->children()
+        $pluginNode = $root->children()
             ->arrayNode('clients')
+                ->fixXmlConfig('plugin')
```

Comment:
this must go after the `->prototype('array')` array line to apply to the things inside the array and not to the array itself (which has numerical keys)